end_to_end_tests/functional_tests/generated_code_execution/test_enums_and_consts.py

```diff
@@ -133,47 +133,27 @@ def test_invalid_values(self, MyModel):
 """
 components:
   schemas:
-    MyEnum:
-      type: string
-      enum: ["a", "b"]
-    MyEnumIncludingNull:
-      type: ["string", "null"]
-      enum: ["a", "b", null]
-    MyNullOnlyEnum:
+    EnumOfNullOnly:
```

Comment:
The changes in this file are just to cut out some test cases that I moved into `test_unions.py`, because ultimately the issue isn't about the enum itself, it's about the union type (created either explicitly with `oneOf`, or explicitly by adding "null" as an allowable type).